### PR TITLE
Update primary-submission-walkthrough.md

### PR DESCRIPTION
**Pull request by Bento007**
- Adding instructions to create a JWT using the HCA CLI. 
- Removed the instruction for using a google_service_account to generate a JWT.

### CODE REVIEW

File: docs/primary-submission-walkthrough.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
+`$ hca dss login`
+
+If you're authenticating on a remote machine use:
+
+`$ hca dss login --remote`
+
+Add the JWT to your environment:
+
+`$ export DCP_TOKEN=$(ython -c 'from hca.dss import DSSClient as c; print(c().get_authenticated_session().token["access_token"])')`
```
Comment (rolando-ebi) [inline docs/primary-submission-walkthrough.md]: `DCP_TOKEN=$(ython -c` should be "python"?

File: docs/primary-submission-walkthrough.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
-You must have a valid token identifying your service in order to use the Ingest API.
+You must have a valid [JWT](https://jwt.io/introduction/) to identifying your service in order to use the Ingest API. A JWT can be created using the [HCA-CLI](https://github.com/HumanCellAtlas/dcp-cli). install with:
 
-The DCP uses Google service accounts for authenticating service-to-service requests. To request a service account and associated RSA key-pair, create an issue in [Fusillade](https://github.com/HumanCellAtlas/fusillade), the DCP's auth service.
+`$ pip install hca`
 
-Use [this](https://github.com/HumanCellAtlas/dcplib/blob/master/dcplib/security/dcp_service_account.py) library with your key file to sign a JWT token identifying your service.
+Once installed authenticate your service by running the command and following the instructions:
 
+`$ hca dss login`
```
Comment (sampierson) [inline docs/primary-submission-walkthrough.md]: Why are we authenticating against the Data Store to ingest things to Ingest?  That makes no sense.  Either you're authenticating against Ingest, in which case the command should be `hca ingest login` or more likely you're authenticating against the entire DCP, in which case the command should be `hca dcp login`.   That's still not entirely intuitive - most users probably have no concept of the different between "HCA" and "DCP", but it's sure better than `dss`.

File: docs/primary-submission-walkthrough.md [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+
+If you're authenticating on a remote machine use:
+
+`$ hca dss login --remote`
```
Comment (sampierson) [inline docs/primary-submission-walkthrough.md]: What do you mean "remote machine"?   Remote to what?  Remote with respect to the DCP?  Remote from my office?

File: docs/primary-submission-walkthrough.md (hunk 1 above)
Comment (Bento007) [reply docs/primary-submission-walkthrough.md]: fixed

File: docs/primary-submission-walkthrough.md (hunk 2 above)
Comment (Bento007) [reply docs/primary-submission-walkthrough.md]: ingest is not integrated into the HCA-CLI and we don't have a generic `hca dcp login`. I will create a ticket to modify this line to `hca ingest login` once ingest is integrated into the HCA CLI, and a ticket ( https://github.com/HumanCellAtlas/dcp-cli/issues/437 )to integrate ingest's API into the HCA CLI.

I think the login for the HCA CLI should be `hca login`.

File: docs/primary-submission-walkthrough.md (hunk 3 above)
Comment (Bento007) [reply docs/primary-submission-walkthrough.md]: I modified statement to be more specific

File: docs/primary-submission-walkthrough.md (hunk 2 above)
Comment (aaclan-ebi) [reply docs/primary-submission-walkthrough.md]: +1 for `hca login`  There should be no need for the user to authenticate for each components.

File: docs/primary-submission-walkthrough.md (hunk 3 above)
Comment (sampierson) [reply docs/primary-submission-walkthrough.md]: 👍

File: docs/primary-submission-walkthrough.md (hunk 2 above)
Comment (sampierson) [reply docs/primary-submission-walkthrough.md]: @Bento007 you're right, `hca login` is the right answer.
We can divide what the UX is, and what the code is doing, here.
Could we not, right now, change the UX to be `hca login` and still use the DSS endpoint, under the covers?

Comment (Bento007) [reply docs/primary-submission-walkthrough.md]: Adding ticket for `hca login` https://github.com/HumanCellAtlas/dcp-cli/issues/439

Comment (Bento007) [reply docs/primary-submission-walkthrough.md]: adding ticket to update documentation https://github.com/HumanCellAtlas/ingest-core/issues/77

**Pull request merged**